bindings/haskell/README.md

```diff
@@ -34,3 +35,15 @@ If you don't want to install `libopendal_hs`, you need to specify library path m
 ```bash
 LIBRARY_PATH=... cabal build
 ```
+
+## Test
+
+```bash
+LD_LIBRARY_PATH=... cabal test
+```
+
+## Doc
```

Comment:
Maybe we should provide a way to visit the docs here?
Just tell user to generate the doc, it's not suitable here.

We can do the same with other bindings. It can be done in another PR.

Reply:
Yeah, just like this PR https://github.com/apache/incubator-opendal/pull/2549/files#diff-9cf2000c53760d837a449f874e53f792819108d3a4bf346336d0f7d082deae2c, add GitHub action for generate and publish doc to GitHub pages.

Reply:
I have added some explanations about visiting documents in the new commit. About github action for it, I'd like to complete it in another PR.